client/src/layers/TextLayer.ts

```diff
@@ -11,32 +11,70 @@ export interface TextData {
   y: number;
   offsetY?: number;
   offsetX?: number;
+  currentPair?: boolean;
 }
 
 export type FormatTextRow = (track: FrameDataTrack) => TextData[] | null;
 
 interface TextLayerParams {
   formatter?: FormatTextRow;
 }
-
 /**
+ * @param track - standard frameDataTrack info
+ * @param additionalNum Number of additional pairs to display, top pair is the current pair
+ * followed by more in descending order.
  * @returns value or null.  null indicates that the text should not be displayed.
  */
-function defaultFormatter(track: FrameDataTrack): TextData[] | null {
+function defaultFormatter(track: FrameDataTrack, additionalNum = 0): TextData[] | null {
```

Comment:
`additionalNum` is sort of an awkward way of specifying a max value.  Does it mean 1 will always be shown, plus some `additionalNum`? 

Could we just make this `maxLines` or `maxPairs` and default it to 1?  I think that's clearer, then you wouldn't have to do `(maxPairs + 1)` below (which is odd anyway because it means that maxPairs = 0 will show 1 line)

Reply:
I think restructuring this will make it clearer by moving and renaming `maxPairs` to `maxAdditionalPairs` down closer to where it is drawn instead of with the default type:confidence pair.